cypress/integration/pages/onDemandAudio/tests.js

```diff
@@ -52,7 +53,8 @@ export default ({ service, pageType, variant, isAmp }) => {
         },
         () => {
           it('should be displayed if the toggle is on, and shows the expected number of items', function test() {
-            // Reload for retry if data didn't update on page
+            // Reload for retry if data didn't update on page, after a wait for next load to have warmed cache
+            cy.wait(5000);
```

Comment:
Hey @LilyL0u , I think we can be a tiny bit smarter with the waits.

Not sure of the exact syntax but an extension on this line: https://github.com/bbc/simorgh/pull/9021/files#diff-b3aaf44c3d2be810fe8b48b205a0c77527f8c8c0795a76bf6227124443a7de16R154

```javascript
// get length and wait if the assertion fails
cy.get('[data-e2e=recent-episodes-list-item]')
  .its('length')
  .then(length => { 
     if(length !== expectedNumberOfEpisodes) {  
        cy.wait(5000); 
     } 
   });

// perform original assertion to fail this test, the next retry should pass 
// as time has been allowed for the upstream cache to populate
cy.get('[data-e2e=recent-episodes-list-item]')
  .its('length')
  .should('eq', expectedNumberOfEpisodes);
```

Reply:
I'll make this change 👍

Reply:
Change made